RuddockWebsite/routes.py

```diff
@@ -0,0 +1,363 @@
+from flask import request, session, g, redirect, url_for, abort, \
+    render_template, flash
+from sqlalchemy import text
+import datetime
+import re
+
+from RuddockWebsite import app, auth
+from RuddockWebsite.constants import *
+from RuddockWebsite.common_helpers import *
+from RuddockWebsite.decorators import *
```

Comment:
Good point, I agree. I was originally planning to fix this in a separate PR, but it's simple enough to do now.